### ISSUE TITLE
Make file run on startup

### ISSUE THREAD

**Issue by StroobWasTaken** [UnamSanctam/UnamBinder#60]
Is there a way i can make the binded files run on startup

**Comment by UnamSanctam** [comment]
No it doesn't have any startup/installation.

**Comment by StroobWasTaken** [comment]
> No it doesn't have any startup/installation.

well, i found this repo https://github.com/f4rber/Dropper 
Would it work if i upload the file to discord and copy the download link into that program

**Comment by UnamSanctam** [comment]
Sure, though it's definitely not the most well made.

**Comment by StroobWasTaken** [comment]
Do you know anything better

**Comment by UnamSanctam** [comment]
Not really, it's especially hard to find any that supports running administrator files on Startup (for example to add Windows Defender exclusions) since doing that is a lot more complicated.

**Comment by StroobWasTaken** [comment]
your SilentCryptoMiner has the option to disable windows defender right, i ran the miner once and now my defender is completely broken xd but i dont mind since i always turn it off

**Comment by UnamSanctam** [comment]
Yes it can add exclusions to Windows Defender (and some version did more than just exclusions), and in the new version it also removes some other similar tools like MSRT.
